Container로 백엔드 실행 시 오류 발생

# What
```
2023-01-16 22:34:00 org.springframework.context.ApplicationContextException: Unable to start web server; nested exception is org.springframework.boot.web.server.WebServerException: Unable to start embedded Tomcat
2023-01-16 22:34:00     at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.onRefresh(ServletWebServerApplicationContext.java:163) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:577) ~[spring-context-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:145) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:754) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:434) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.boot.SpringApplication.run(SpringApplication.java:338) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at com.js.freeproject.FreeProjectApplication.main(FreeProjectApplication.java:13) ~[classes!/:na]
2023-01-16 22:34:00     at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
2023-01-16 22:34:00     at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[na:na]
2023-01-16 22:34:00     at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[na:na]
2023-01-16 22:34:00     at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[na:na]
2023-01-16 22:34:00     at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49) ~[app.jar:na]
2023-01-16 22:34:00     at org.springframework.boot.loader.Launcher.launch(Launcher.java:108) ~[app.jar:na]
2023-01-16 22:34:00     at org.springframework.boot.loader.Launcher.launch(Launcher.java:58) ~[app.jar:na]
2023-01-16 22:34:00     at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:88) ~[app.jar:na]
2023-01-16 22:34:00 Caused by: org.springframework.boot.web.server.WebServerException: Unable to start embedded Tomcat
2023-01-16 22:34:00     at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.initialize(TomcatWebServer.java:142) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.<init>(TomcatWebServer.java:104) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory.getTomcatWebServer(TomcatServletWebServerFactory.java:450) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory.getWebServer(TomcatServletWebServerFactory.java:199) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.createWebServer(ServletWebServerApplicationContext.java:182) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.onRefresh(ServletWebServerApplicationContext.java:160) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     ... 14 common frames omitted
2023-01-16 22:34:00 Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jwtFilter' defined in URL [jar:file:/app.jar!/BOOT-INF/classes!/com/js/freeproject/global/jwt/JwtFilter.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userService': Lookup method resolution failed; nested exception is java.lang.IllegalStateException: Failed to introspect Class [com.js.freeproject.domain.user.application.UserService] from ClassLoader [org.springframework.boot.loader.LaunchedURLClassLoader@2e0fa5d3]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:800) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:229) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1372) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1222) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:582) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:410) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1352) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1195) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:582) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:213) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.boot.web.servlet.ServletContextInitializerBeans.getOrderedBeansOfType(ServletContextInitializerBeans.java:212) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.boot.web.servlet.ServletContextInitializerBeans.getOrderedBeansOfType(ServletContextInitializerBeans.java:203) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.boot.web.servlet.ServletContextInitializerBeans.addServletContextInitializerBeans(ServletContextInitializerBeans.java:97) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.boot.web.servlet.ServletContextInitializerBeans.<init>(ServletContextInitializerBeans.java:86) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.getServletContextInitializerBeans(ServletWebServerApplicationContext.java:260) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.selfInitialize(ServletWebServerApplicationContext.java:234) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.springframework.boot.web.embedded.tomcat.TomcatStarter.onStartup(TomcatStarter.java:53) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5219) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1396) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1386) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at java.base/java.util.concurrent.FutureTask.run(Unknown Source) ~[na:na]
2023-01-16 22:34:00     at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at java.base/java.util.concurrent.AbstractExecutorService.submit(Unknown Source) ~[na:na]
2023-01-16 22:34:00     at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:919) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:835) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1396) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1386) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at java.base/java.util.concurrent.FutureTask.run(Unknown Source) ~[na:na]
2023-01-16 22:34:00     at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at java.base/java.util.concurrent.AbstractExecutorService.submit(Unknown Source) ~[na:na]
2023-01-16 22:34:00     at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:919) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:263) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.apache.catalina.core.StandardService.startInternal(StandardService.java:432) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:927) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.apache.catalina.startup.Tomcat.start(Tomcat.java:486) ~[tomcat-embed-core-9.0.53.jar!/:na]
2023-01-16 22:34:00     at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.initialize(TomcatWebServer.java:123) ~[spring-boot-2.5.5.jar!/:2.5.5]
2023-01-16 22:34:00     ... 19 common frames omitted
2023-01-16 22:34:00 Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userService': Lookup method resolution failed; nested exception is java.lang.IllegalStateException: Failed to introspect Class [com.js.freeproject.domain.user.application.UserService] from ClassLoader [org.springframework.boot.loader.LaunchedURLClassLoader@2e0fa5d3]
2023-01-16 22:34:00     at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.determineCandidateConstructors(AutowiredAnnotationBeanPostProcessor.java:289) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.determineConstructorsFromBeanPostProcessors(AbstractAutowireCapableBeanFactory.java:1302) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1219) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:582) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1380) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1300) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:887) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:791) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     ... 68 common frames omitted
2023-01-16 22:34:00 Caused by: java.lang.IllegalStateException: Failed to introspect Class [com.js.freeproject.domain.user.application.UserService] from ClassLoader [org.springframework.boot.loader.LaunchedURLClassLoader@2e0fa5d3]
2023-01-16 22:34:00     at org.springframework.util.ReflectionUtils.getDeclaredMethods(ReflectionUtils.java:481) ~[spring-core-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.util.ReflectionUtils.doWithLocalMethods(ReflectionUtils.java:321) ~[spring-core-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.determineCandidateConstructors(AutowiredAnnotationBeanPostProcessor.java:267) ~[spring-beans-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     ... 81 common frames omitted
2023-01-16 22:34:00 Caused by: java.lang.NoClassDefFoundError: com/sun/jdi/request/DuplicateRequestException
2023-01-16 22:34:00     at java.base/java.lang.Class.getDeclaredMethods0(Native Method) ~[na:na]
2023-01-16 22:34:00     at java.base/java.lang.Class.privateGetDeclaredMethods(Unknown Source) ~[na:na]
2023-01-16 22:34:00     at java.base/java.lang.Class.getDeclaredMethods(Unknown Source) ~[na:na]
2023-01-16 22:34:00     at org.springframework.util.ReflectionUtils.getDeclaredMethods(ReflectionUtils.java:463) ~[spring-core-5.3.10.jar!/:5.3.10]
2023-01-16 22:34:00     ... 83 common frames omitted
2023-01-16 22:34:00 Caused by: java.lang.ClassNotFoundException: com.sun.jdi.request.DuplicateRequestException
2023-01-16 22:34:00     at java.base/java.net.URLClassLoader.findClass(Unknown Source) ~[na:na]
2023-01-16 22:34:00     at java.base/java.lang.ClassLoader.loadClass(Unknown Source) ~[na:na]
2023-01-16 22:34:00     at org.springframework.boot.loader.LaunchedURLClassLoader.loadClass(LaunchedURLClassLoader.java:151) ~[app.jar:na]
2023-01-16 22:34:00     at java.base/java.lang.ClassLoader.loadClass(Unknown Source) ~[na:na]
2023-01-16 22:34:00     ... 87 common frames omitted
2023-01-16 22:34:00
```

# Why
`import com.sun.jdi.request.DuplicateRequestException;`
`if (finduser != null) { throw new DuplicateRequestException(user.getEmail()); }`
다음과 같이 사용자 생성을 할 때, 이미 존재하는 사용자일 경우 `DuplicateRequestException`을 발생 시켰다.
하지만 `com.sun.jdi` 패키지는 디버깅에 사용되는 패키지라고 한다...

# How
해당 `DuplicateRequestException`을 다른 예외를 발생 시키게 한다.

--- a/backend/src/main/java/com/js/freeproject/domain/user/application/UserService.java
+++ b/backend/src/main/java/com/js/freeproject/domain/user/application/UserService.java
@@ -20,7 +20,6 @@ import com.js.freeproject.domain.user.dto.UserRequest;
 import com.js.freeproject.global.jwt.CustomUserDetails;
 import com.js.freeproject.global.util.MailUtil;
 import com.js.freeproject.global.util.RedisUtil;
-import com.sun.jdi.request.DuplicateRequestException;
 
 import io.lettuce.core.RedisCommandExecutionException;
 import javassist.NotFoundException;
@@ -42,7 +41,7 @@ public class UserService {
         User finduser = userRepo.findByEmail(user.getEmail());
         String userImage = "https://freepjt.s3.ap-northeast-2.amazonaws.com/user/mainCharachter.png";
         if (finduser != null) {
-            throw new DuplicateRequestException(user.getEmail());
+            throw new IllegalArgumentException(user.getEmail());
         }
         if (user.getImage() != null) {
             userImage = s3Service.uploadImage(user.getImage(), "user");

--- a/backend/src/main/java/com/js/freeproject/domain/user/controller/UserController.java
+++ b/backend/src/main/java/com/js/freeproject/domain/user/controller/UserController.java
@@ -27,7 +27,6 @@ import com.js.freeproject.domain.user.dto.UserResponse;
 import com.js.freeproject.global.jwt.CustomUserDetails;
 import com.js.freeproject.global.jwt.TokenProvider;
 import com.js.freeproject.global.util.RedisUtil;
-import com.sun.jdi.request.DuplicateRequestException;
 
 import io.lettuce.core.RedisCommandExecutionException;
 import io.swagger.annotations.Api;
@@ -96,7 +95,7 @@ public class UserController {
 		try {
 			user.setPass(passwordEncoder.encode(user.getPass()));
 			userService.createUser(user);
-		} catch(DuplicateRequestException e) {
+		} catch(IllegalArgumentException e) {
 			log.info("이미 가입된 이메일입니다.");
 			return ResponseEntity.status(401).body(CommonResponse.of("이미 가입된 이메일입니다."));
 		} catch(Exception e) {

